COMP: Fix compiler warnings in ComputeSurfaceFeaturesTest

imikejackson opened the pull request.

imikejackson committed:
COMP: Fix compiler warnings in ComputeSurfaceFeaturesTest

imikejackson committed:
COMP: Fix compiler warnings in ITKThresholdMaximumConnectedComponentsImageFilter

JDuffeyBQ commented on src/Plugins/SimplnxCore/test/ComputeSurfaceFeaturesTest.cpp:
```diff
@@ -147,7 +147,8 @@ TEST_CASE("SimplnxCore::ComputeSurfaceFeaturesFilter: 3D", "[SimplnxCore][Comput
     for(usize i = 0; i < featureArrayExemplary.getSize(); i++)
     {
-      REQUIRE(featureArrayExemplary[i] == createdFeatureArray[i]);
+      uint8 value = featureArrayExemplary[i] == true ? 1 : 0;
```
```suggestion
      uint8 value = featureArrayExemplary[i] ? 1 : 0;
```

imikejackson committed:
Update src/Plugins/SimplnxCore/test/ComputeSurfaceFeaturesTest.cpp

Co-authored-by: Jared Duffey <jared.duffey@bluequartz.net>

imikejackson committed:
Use the max value for a uint16. This in reality should work for most normal images.

In theory a user could have a data set that uses 64 bit float (or even 32 bit float)
where they might need to bump up the upper boundary value if they data necessitates
that kind of value.

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>
